financialaid/management/commands/generate_exchange_rates_test.py

```diff
@@ -0,0 +1,47 @@
+"""
+Test for management command generating exchange rates
+"""
+from mock import patch
+
+from financialaid.constants import CURRENCY_EXCHANGE_RATE_API_REQUEST_URL
+from financialaid.management.commands import generate_exchange_rates
+from financialaid.models import CurrencyExchangeRate
+from search.base import ESTestCase
+
+
+@patch('financialaid.tasks.requests.get')
+class GenerateExchangeRatesTest(ESTestCase):
+    """
+    Tests for generate_exchange_rates management command
+    """
+    @classmethod
+    def setUpTestData(cls):
+        cls.command = generate_exchange_rates.Command()
+
+    def setUp(self):
+        super(GenerateExchangeRatesTest, self).setUp()
+        self.data = {
+            "extraneous information": "blah blah blah",
+            "rates": {
+                "CBA": "3.5",
+                "FED": "1.9",
+                "RQP": "0.5"
+            }
+        }
+
+    def test_currency_exchange_rate_command(self, mocked_request):
+        """
+        Assert currency exchange rates are created using management command
+        """
+        mocked_request.return_value.json.return_value = self.data
+        assert CurrencyExchangeRate.objects.count() == 0
+        self.command.handle("generate_exchange_rates")
+        called_args, _ = mocked_request.call_args
+        assert called_args[0] == CURRENCY_EXCHANGE_RATE_API_REQUEST_URL
+        assert CurrencyExchangeRate.objects.count() == 3
+        currency = CurrencyExchangeRate.objects.get(currency_code="CBA")
+        assert currency.exchange_rate == 3.5
+        currency = CurrencyExchangeRate.objects.get(currency_code="FED")
+        assert currency.exchange_rate == 1.9
+        currency = CurrencyExchangeRate.objects.get(currency_code="RQP")
+        assert currency.exchange_rate == 0.5
```

Comment:
Also, I'm not familiar with the details of the test framework here. Is there something that will automatically clean up or rollback the objects that this test creates in the database? If not, it's a good idea to write a `cleanUp()` method that removes the objects that the test created, so that the tests are independent of each other.